service/debugger/locations.go

```diff
@@ -0,0 +1,291 @@
+package debugger
+
+import (
+	"debug/gosym"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/derekparker/delve/service/api"
+)
+
+const maxFindLocationCandidates = 5
```

Comment:
This seems to be only used in NormalLocationSpec.Find which returns an error if > 1 locations are found, is this necessary?